application/common/config/main.php

```diff
@@ -141,6 +142,9 @@
             'subjectForMfaDisabled' => Env::get('SUBJECT_FOR_MFA_DISABLED'),
             'subjectForMfaManager' => Env::get('SUBJECT_FOR_MFA_MANAGER'),
             'subjectForMfaManagerHelp' => Env::get('SUBJECT_FOR_MFA_MANAGER_HELP'),
+            'subjectForMfaRecovery' => Env::get('SUBJECT_FOR_MFA_RECOVERY'),
+            'subjectForMfaRecoveryHelp' => Env::get('SUBJECT_FOR_MFA_RECOVERY_HELP'),
```

Comment:
Even though we've used environment variables for subject customization in the past, I don't think we should this time. For one, only one IdP instance is going to use this option. And we almost have never actually used custom subject text.

Reply:
I agree 👍

Reply:
I don't know if it's worth it, but if we want to maintain backwards compatibility for this, I think we would need to fallback to the `SUBJECT_FOR_MFA_MANAGER` / `SUBJECT_FOR_MFA_MANAGER_HELP` env. vars.
```suggestion
            'subjectForMfaRecovery' => Env::get(
                'SUBJECT_FOR_MFA_RECOVERY',
                Env::get('SUBJECT_FOR_MFA_MANAGER')
            ),
            'subjectForMfaRecoveryHelp' => Env::get(
                'SUBJECT_FOR_MFA_RECOVERY_HELP',
                Env::get('SUBJECT_FOR_MFA_MANAGER_HELP')
            ),
```
Again, I don't know if it's worth this, especially given that we are moving away from these customizable subject lines (due to complexity and lack of use).